### DISCUSSION TITLE
Welcome to MyReactApp Discussions!

### DISCUSSION THREAD

**Discussion by elliotmarks** [elliotmarks/MyReactApp#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elliotmarks** [comment]
How can we set up react application?

**Comment by jigarvakil** [comment, marked as answer]
Hi @elliotmarks you can use npx create-react-app my-app for creating react application and if you want to create react application using typescript then you can use npx create-react-app my-app --template typescript.
Thanks

**Reply by elliotmarks** [reply]
This works Thanks!!!

**Comment by elliotmarks** [comment]
@jigarvakil one more question which is good YARN or NPM?

**Comment by jigarvakil** [comment]
@elliotmarks 
As previously stated, Yarn installs dependency packages in parallel, whereas NPM installs them sequentially. As a result, Yarn outperforms NPM when installing bigger files. Both tools can save dependent files to the offline cache.
you can refer this link : https://www.imaginarycloud.com/blog/npm-vs-yarn-which-is-better/#:~:text=As%20previously%20stated%2C%20Yarn%20installs,files%20to%20the%20offline%20cache.
